panda/src/gobj/texturePool.h

```diff
@@ -175,6 +185,10 @@ class EXPCL_PANDA_GOBJ TexturePool {
     Filename _alpha_fullpath;
     int _primary_file_num_channels = 0;
     int _alpha_file_channel = 0;
+    int _texture_format;
+    int _texture_quality;
+    int _texture_compress;
```

Comment:
Any particular reason these are ints instead of the appropriate enum from texture.h?

Reply:
The enums are defined in eggTexture.h. If I add that header file to LoaderOptions.h doesn't that cause the LoaderOptions to now be required in the core when it is supposed to be optional?

Reply:
I was talking about the enums in texture.h, but I realised that that's in gobj whereas LoaderOptions is in putil, so you're right that that wouldn't work unless we moved the enums.

Reply:
I am not sure if that is worth doing. The values are for comparison only the way I use them. However, others might have a use for them.

Reply:
Any idea why the Buildsystem checks are failing?

Reply:
I have some time this weekend. Let me know if there is more to do on this.

Reply:
The build failures are unrelated to your changes; if you merge in from latest master they should go away.